plugins/providers/virtualbox/synced_folder.rb

```diff
@@ -124,8 +124,10 @@ def share_folders(machine, folders, transient)
               name: os_friendly_id(id),
               hostpath: hostpath.to_s,
               transient: transient,
-              SharedFoldersEnableSymlinksCreate: enable_symlink_create
-            }
+              SharedFoldersEnableSymlinksCreate: enable_symlink_create,
+              automount: data[:automount]
+            }.delete_if { |_,v| v.nil?}
```

Comment:
I think we can remove this `delete_if`. If we want to prevent a `nil` value and force either a `true` or `false` type, we can just double negate the value `!!data[:automount]`.